mantidproject/mantid
# Loader for D11B and D22B

**Description of work.**
This PR adds a loader for ILL's SANS instruments D11B and D22B, which are upgraded versions of D11 and D22.

**To test:**
- Load `ExternalData/Testing/Data/UnitTest/ILL/D11/027194.nxs` and in the instrument view, check that the instrument seems correct.
- Load `ExternalData/Testing/Data/UnitTest/ILL/D22/375508.nxs` and in the instrument view, check that it looks correct (there is no data on the right panel, this is normal).


Part of #26906 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
